mongodbatlas/resource_mongodbatlas_custom_db_role.go

```diff
@@ -331,23 +332,21 @@ func flattenActionResources(resources []matlas.Resource) []map[string]interface{
 }
 
 func expandInheritedRoles(d *schema.ResourceData) []matlas.InheritedRole {
-	var inheritedRoles []matlas.InheritedRole
+	vIR := d.Get("inherited_roles").(*schema.Set).List()
```

Comment:
I would propose to use speaking variable names.

Reply:
I can change this more clarify but in golang have this kind of variables is very common, it's a convention write short variable names.